server/package.json

```diff
@@ -27,17 +28,17 @@
     "passport-jwt": "4.0.0",
     "passport-local": "1.0.0",
     "winston": "3.3.3",
-    "winston-cloudwatch": "0.9.0",
+    "winston-cloudwatch": "3.1.1",
     "yup": "0.32.11"
   },
   "devDependencies": {
-    "eslint": "7.7.0",
+    "eslint": "^7.32.0",
```

Comment:
I think `^` was introduced by error?

Reply:
Same reason as above

Reply:
Husky should be fine, I am not sure if any future version of ES lint might introduce breaking changes. But for a small project, I hope it should be fine.